lib/conway.dart

```diff
@@ -122,6 +121,25 @@ class WorldState {
     }
     return buffer.toString();
   }
+
+  @override
+  bool operator ==(dynamic other) {
```

Comment:
I can't recall if Dart requires overriding "hash" at the same time as == or not?